common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3215,11 +3225,6 @@
           "<variableName> is a VARIABLE and cannot be updated using the SET statement. Use SET VARIABLE <variableName> = ... instead."
         ]
       },
-      "TABLE_OPERATION" : {
-        "message" : [
-          "Table <tableName> does not support <operation>. Please check the current catalog and namespace to make sure the qualified table name is expected, and also check the catalog implementation which is configured by \"spark.sql.catalog\"."
```

Comment:
Let's not degrade the error message. I think there are two kinds if unsupported table operations: 1) the table implementation does not support certain DS v2 features. 2) it's a view not table.

Now my preference is to keep this sub error class here, but add two new error class `EXPECT_VIEW_GOT_TABLE` and `EXPECT_TABLE_GOT_VIEW` to replace the `UNSUPPORTED_TABLE_OPERATION` and `UNSUPPORTED_VIEW_OPERATION`

Reply:
Let me refactor it.

Reply:
Done.
PS: After the pr, I will open a new seperate pr to merge `_LEGACY_ERROR_TEMP_1113` into `UNSUPPORTED_FEATURE.TABLE_OPERATION`.